### ISSUE TITLE
Support older Android versions

### ISSUE THREAD

**Issue by Pyrdacor** [dotnet/Silk.NET#812]
# Summary of feature

Support older Android version.

# Comments

I am not sure if this is a bug or a missing feature. Silk.NET seems to require Android 11 (Target Framework net6.0-android30.0 or above). You also say so in the readme. However many mobile devices out there still use older Android versions and can't upgrade to Android 11 (including my own one). Genymotion also brings only up to 10 with it.

Is there a reason to limit it to 11+?

**Comment by Perksey** [comment]
The Android 30 TFM is just the default. The TFM doesn't affect your application's minimum SDK version, you can still specify a lower minimum using your android manifest. So not much impact and not really anything stopping you from targeting older devices so long as you don't use any APIs not in your desired API level.

**Comment by Pyrdacor** [comment]
Thanks for clarification. Already tried that but unfortunately the app can't be installed on my phone. Thought that was the reason. But maybe it's not. So I have to find the real reason.

**Comment by roeyskoe** [comment]
Have you looked at [Android Demos]( https://github.com/dotnet/Silk.NET/tree/main/examples/CSharp/OpenGL%20Demos)?
They were made with my Android 10 phone.

**Comment by Pyrdacor** [comment]
Currently I can't test anything on a real phone or an emulator. I managed to test it once but after I restarted VS2022, it never managed to start the emulator again. It just hangs there forever (even after 16 hours). Testing with real phone doesn't work either. I tested one two systems but no luck yet. VS2022 seems quite unstable and the Android support isn't very good as well. The emulator is slow as hell and everytime I got another error which wasn't there in the first place. Sometimes restarting VS helps, sometimes not. It is very frustrating and I wasted a few days now only to test this. And I am still trying.
